targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC/gpio_irq_api.c

```diff
@@ -25,6 +25,8 @@
 #include "fsl_pint.h"
 #include "mbed_error.h"
 
+#define INTERRUPT_PORTS 2
```

Comment:
I suppose all LPC targets only have two ports that are interruptible?

Reply:
Yes.